### PR TITLE
test: Additional unit tests for event timer, Inizio and Google Analytics

### PR DESCRIPTION
**Pull request by chrislomaxjones**
## What does this change?

This PR adds:
- Additional unit tests to EventTimer, testing multiple events and `mark`.
- Additional unit tests to the Inizio third party tag.
- Unit tests for google analytics.

## Why?

Increase code coverage!

### CODE REVIEW

File: src/EventTimer.spec.ts [hunk 1]
```diff
@@ -6,27 +6,27 @@ jest.mock('./GoogleAnalytics', () => ({
...
+			mark: jest.fn(),
+			getEntriesByName: jest.fn().mockReturnValue([
+				{
+					duration: 1,
+					entryType: 'mark',
+					name: 'commercial event',
+					startTime: 1,
+				},
+			]),
+		};
```
Comment (mxdvl) [inline src/EventTimer.spec.ts:21]: This `const performance` can still live outside of the `beforeEach`. You don’t need to redefine this variable each time, only assign to `window.performance`.

*not a blocker*

File: src/third-party-tags/inizio.ts [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-		survey: { measurementId: string },
-	) => {
-		if (surveyAvailable) {
-			if (window.googletag) {
-				window.googletag.cmd.push(() => {
-					window.googletag?.pubads().setTargeting('inizio', 't');
-				});
-			}
-			console.log(`surveyAvailable: ${survey.measurementId}`);
+export const handleQuerySurveyDone = (
```
Comment (mxdvl) [inline src/third-party-tags/inizio.ts:3]: The convention for testing is to export an object called `_` (underscore) at the end of the file, like so:

```ts
export const _ = {
  handleQuerySurveyDone,
  onLoad,
}
```

See some examples here:
- https://github.com/guardian/frontend/blob/40c9d66bf4832b9aef7ca389db535b5e282f9185/static/src/javascripts/projects/common/modules/commercial/geo-utils.js#L26
- https://github.com/guardian/consent-management-platform/blob/2edc662cf92880cd38ad0e1d02e5a997baddf8df/src/onConsentChange.ts#L60

**Review by mxdvl — APPROVED** [review]
Excellent, our coverage is now 💯

File: src/third-party-tags/inizio.ts (hunk 2 above)
Comment (chrislomaxjones) [reply src/third-party-tags/inizio.ts]: Great, good to know!

**Comment by mxdvl** [comment]
Merge away!

**Pull request merged**